package.json

```diff
@@ -90,22 +90,24 @@
     "sitemap-static": "^0.4.2",
     "style-loader": "^0.18.2",
     "tap-min": "^1.2.1",
-    "tap-parser": "^5.4.0",
+    "tap-parser": "^6.0.1",
     "through2": "^2.0.3",
     "url-loader": "^0.5.9",
-    "webpack": "^3.0.0",
-    "webpack-dev-server": "^2.5.0",
+    "webpack": "^3.5.5",
+    "webpack-dev-server": "^2.7.1",
     "webpack-merge": "^4.1.0",
     "yaml-frontmatter-loader": "^0.1.0"
   },
   "dependencies": {
-    "preact": "^8.1.0",
-    "preact-compat": "3.16.0",
+    "ajv": "^5.2.2",
+    "preact": "^8.2.5",
+    "preact-compat": "3.17.0",
     "prop-types": "^15.5.10",
     "react": "^15.6.1",
     "react-dom": "^15.6.1",
-    "react-router": "^4.1.1",
-    "tool-list": "^0.10.1",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
+    "tool-list": "^0.11.0",
     "whatwg-fetch": "^2.0.3"
   }
 }
```

Comment:
Thanks for updating all the outdated dependencies and taking care of the yarn file. We aren't really too systematic about updating yarn, I would think it should be on release but seeing as we don't really have a formalized process (just fires when master is updated) this should be fine.

By the way, why add `ajv`? Not seeing that used anywhere.

Reply:
This is warning of `yarn`: 
```
warning "ajv-keywords@1.5.1" has incorrect peer dependency "ajv@>=4.10.0".
```
I really do not see this as a necessity, but to be careful I add it.